README.md

```diff
@@ -628,6 +628,7 @@ Name | Website | Region
 [Webikon](/company-profiles/webikon.md) | https://www.webikon.sk/en/ | Worldwide
 [Wells Fargo](/company-profiles/wells-fargo.md) | https://www.wellsfargo.com/ |
 [wemake.services](/company-profiles/wemake-services.md) | https://wemake.services/ | Worldwide
+[WeMakeMVP](/company-profiles/wemakemvp.md) | https://www.wemakemvp.com/ | Ukraine & Worldwide
```

Comment:
Ukraine or Worldwide?

Reply:
Changed to Worldwide, our company is ready to accept people from all around the world. The thing is that founders are based in Ukraine.